dask/dataframe/methods.py

```diff
@@ -371,7 +372,12 @@ def size(x):
 
 
 def values(df):
-    return df.values
+    values = df.values
```

Comment:
This is better than throwing an exception. I'd add a warning here. Something like this:

```
warnings.warn('Dask doesn't have support for X dtype yet. Values were converted from X to object. See issue XXX for more information', UserWarning)
```

Users might notice the conversion and think it's a bug, that would let them know that there's no need to file an issue.

Reply:
Agree with @j-bennet that casting to object is better than an error, and that we should probably warn the user.

Reply:
Yeah, agreed that's the most friendly thing to do. I pushed a commit yesterday which adds a warning about converting to `object` dtype in this case